precompiled-token/test/PrecompiledToken.test.ts

```diff
@@ -0,0 +1,50 @@
+import { expect, use } from 'chai';
+import { solidity } from 'ethereum-waffle';
+import { Contract } from 'ethers';
+import ADDRESS from "@acala-network/contracts/utils/Address"
+
+import { evmChai, Signer, TestProvider } from '@acala-network/bodhi';
+import { WsProvider } from '@polkadot/api';
+
+const Token = require('@acala-network/contracts/build/contracts/Token.json');
+
+use(solidity);
+use(evmChai);
+
+const provider = new TestProvider({
```

Comment:
here we can also reuse the util from this PR, to be more DRY https://github.com/AcalaNetwork/waffle-tutorials/pull/2/files#diff-7583809507a13391057c3aee722e422d50d961a87e2a3dbf05ea492dc6465c94R8